### ISSUE TITLE
Missing logical or operator

### ISSUE THREAD

**Issue by mxmlnkn** [michaelrsweet/abnf#6]
When using a the ABNF reduced from [here](https://tools.ietf.org/html/rfc5954#section-4.1) as input:
```ABNF
dec-octet = DIGIT / 2*DIGIT / "1" 2DIGIT
```
I get
```regex
^([0-90-9]{2,}|1[0-9]{2,2})$
```
I'm pretty sure it should be:
```regex
^([0-9]|[0-9]{2,}|1[0-9]{2,2})$
```

**Comment by michaelrsweet** [comment]
This is over-optimization of the regular expression...

**Comment by michaelrsweet** [comment]
[master 2533c06] Fix over-optimization of ABNF (Issue #6)

I'm sure in time there may need to be some tweaks to this, but for now it seems to catch the common cases.
